demisto_sdk/commands/common/logger.py

```diff
@@ -315,82 +367,93 @@ def logging_setup(
     Returns:
         logging.Logger: logger object
     """
-
     if not hasattr(logging.getLoggerClass(), "success"):
         _add_logging_level("SUCCESS", SUCCESS_LEVEL)
 
     global logger
-    global current_log_file_path
-    global log_file_name_notified
+    global LOG_FILE_PATH
+    global LOG_FILE_PATH_PRINT
 
     console_handler = logging.StreamHandler()
     console_handler.set_name(CONSOLE_HANDLER)
     console_handler.setLevel(console_log_threshold or logging.INFO)
 
-    if custom_log_path := os.getenv("DEMISTO_SDK_LOG_FILE_PATH"):
-        current_log_file_path = Path(custom_log_path)
-    else:
-        current_log_file_path = Path(log_file_path or LOG_FILE_PATH)
-        if current_log_file_path.is_dir():
-            current_log_file_path = current_log_file_path / LOG_FILE_NAME
-    file_handler = RotatingFileHandler(
-        filename=current_log_file_path,
-        mode="a",
-        maxBytes=DEMISTO_SDK_LOG_FILE_SIZE,
-        backupCount=DEMISTO_SDK_LOG_FILE_COUNT,
-    )
-    file_handler.set_name(FILE_HANDLER)
-    file_handler.setLevel(file_log_threshold or logging.DEBUG)
-
-    if string_to_bool(os.getenv("DEMISTO_SDK_LOG_NO_COLORS", "False")):
+    if environment_variable_to_bool("DEMISTO_SDK_LOG_NO_COLORS"):
         console_handler.setFormatter(fmt=NoColorFileFormatter())
     else:
         console_handler.setFormatter(fmt=ColorConsoleFormatter())
 
-    file_formatter = NoColorFileFormatter()
-    file_handler.setFormatter(fmt=file_formatter)
+    log_handlers: List[logging.Handler] = [console_handler]
+
+    if log_file_path_str := (log_file_path or os.getenv("DEMISTO_SDK_LOG_FILE_PATH")):
+        log_file_directory_path = Path(log_file_path_str)
 
+        # Can't use 'logger.error' here, as the logger is not yet initialized.
+        if not log_file_directory_path.is_dir():
+            print(  # noqa: T201
+                f"Error: Configured logs path '{log_file_directory_path}' does not exist."
+            )
+            exit(1)
```

Comment:
did you test this part? can we use print here?
maybe instead of print just exit with a message

Reply:
Printing works, I've tested it, but you are right, exiting with an error is probably better. Good suggestion.
```suggestion
            exit(
                f"Error: Configured logs path '{log_file_directory_path}' does not exist."
            )
```